### PR TITLE
gui: add welcome_park as a commercial_app

### PR DESCRIPTION
**Pull request by nishinji**
Eliminate the need to remove NPXS from folder name as currently
image
image

### CODE REVIEW

**Comment by Croden1999** [comment]
could you hide this information? is not same as vita

**Comment by nishinji** [comment]
> could you hide this information? is not same as vita also live area not exist buttons

ah i see you're right

**Comment by Croden1999** [comment]
i remember Mr. Zangetsu is PR can running it, not need change name, just pust the game in pd0. https://github.com/Vita3K/Vita3K/pull/2826

**Comment by Zangetsu38** [comment]
> i remember Mr. Zangetsu's PR can running it, not need change name, just pust this app (decrypt) in pd0. #2826

My old pr is big refactor of app db, with so support app inside pd0 without moving it.

and good works for this pr @nishinji

**Comment by Zangetsu38** [comment]
> could you hide this information? is not same as vita also live area not exist buttons

is right yes, psvita no have any things like info for this app, for it is litteraly like system app

File: vita3k/gui/src/app_context_menu.cpp [hunk 1]
```diff
@@ -307,7 +307,8 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
     auto &common = emuenv.common_dialog.lang.common;
     auto &lang_compat = gui.lang.compatibility;
 
-    const auto is_commercial_app = title_id.find("PCS") != std::string::npos;
+    const auto is_commercial_app = title_id.starts_with("PCS") || title_id == "NPXS10007";
+    const auto is_system_app = title_id.starts_with("NPXS") && !(title_id == "NPXS10007");
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:311]:
```suggestion
    const auto is_system_app = title_id.starts_with("NPXS") && (title_id =! "NPXS10007");
```

File: vita3k/gui/src/app_context_menu.cpp [hunk 2]
```diff
@@ -610,7 +611,7 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
         ImGui::PushTextWrapPos(display_size.x - (85.f * SCALE.x));
         ImGui::TextColored(GUI_COLOR_TEXT, "%s", APP_INDEX->title.c_str());
         ImGui::PopTextWrapPos();
-        if (title_id.find("NPXS") == std::string::npos) {
+        if (!is_system_app && !(title_id == "NPXS10007")) {
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:614]:
```suggestion
        if (!is_system_app && (title_id != "NPXS10007")) {
```

File: vita3k/gui/src/app_context_menu.cpp [hunk 3]
```diff
@@ -307,7 +307,8 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
     auto &common = emuenv.common_dialog.lang.common;
     auto &lang_compat = gui.lang.compatibility;
 
-    const auto is_commercial_app = title_id.find("PCS") != std::string::npos;
+    const auto is_commercial_app = title_id.starts_with("PCS") || title_id == "NPXS10007";
+    const auto is_system_app = title_id.starts_with("NPXS") && title_id != "NPXS10007";
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:311]:
```suggestion
    const auto is_commercial_app = title_id.starts_with("PCS") || (title_id == "NPXS10007");
    const auto is_system_app = title_id.starts_with("NPXS") && (title_id != "NPXS10007");
```

very small change ^^
is formating code

File: vita3k/gui/src/app_context_menu.cpp [hunk 4]
```diff
@@ -610,7 +611,7 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
         ImGui::PushTextWrapPos(display_size.x - (85.f * SCALE.x));
         ImGui::TextColored(GUI_COLOR_TEXT, "%s", APP_INDEX->title.c_str());
         ImGui::PopTextWrapPos();
-        if (title_id.find("NPXS") == std::string::npos) {
+        if (!is_system_app && title_id != "NPXS10007") {
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:614]:
```suggestion
        if (!is_system_app && (title_id != "NPXS10007")) {
```
same :)

**Comment by nishinji** [comment]
> i remember Mr. Zangetsu's PR can running it, not need change name, just pust this app (decrypt) in pd0. #2826

Looks like zang has already been there, so I'll leave it to him :D
It's annoying and I don't have time to do it.

**Comment by Croden1999** [comment]
> Looks like zang has already been there, so I'll leave it to him :D It's annoying and I don't have time to do it.

yes, is annoying 😂

**Comment by Zangetsu38** [comment]
for now, i have no found how install/decrypt pre-install package, ex-fat...

**Review by Zangetsu38 — APPROVED** [review]

**Review by Macdu — APPROVED** [review]

**Pull request merged**